app/views/claims/where_teaching.html.erb

```diff
@@ -10,9 +10,14 @@
           <fieldset class="govuk-fieldset">
 
             <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
-              <h1 class="govuk-fieldset__heading"><%= t("student_loans.questions.employment_status") %></h1>
+              <h1 class="govuk-fieldset__heading"><%= t("student_loans.questions.where_teaching") %></h1>
```

Comment:
The answers (which start with "Yes, ") need to be changed as a result of this change. There's also a `choose_still_teaching` helper, which uses these answers and which should be renamed.

Reply:
Good catch.

Reply:
Done.